example/main.tf

```diff
@@ -52,60 +53,58 @@ resource "google_compute_subnetwork" "vpc_subnetwork" {
   # a dash, lowercase letter, or digit, except the last character, which
   # cannot be a dash.
   #name = "default-${var.gcp_cluster_region}"
-  name = "${var.vpc_subnetwork_name}"
+  name = var.vpc_subnetwork_name
 
-  ip_cidr_range = "${var.vpc_subnetwork_cidr_range}"
+  ip_cidr_range = var.vpc_subnetwork_cidr_range
 
   # The network this subnet belongs to. Only networks that are in the
   # distributed mode can have subnetworks.
-  network = "${var.vpc_network_name}"
+  network = var.vpc_network_name
 
   # An array of configurations for secondary IP ranges for VM instances
   # contained in this subnetwork. The primary IP of such VM must belong to the
   # primary ipCidrRange of the subnetwork. The alias IPs may belong to either
   # primary or secondary ranges.
-  secondary_ip_range = [
-    {
-      range_name    = "${var.cluster_secondary_range_name}"
-      ip_cidr_range = "${var.cluster_secondary_range_cidr}"
-    },
-    {
-      range_name    = "${var.services_secondary_range_name}"
-      ip_cidr_range = "${var.services_secondary_range_cidr}"
-    },
-  ]
+  secondary_ip_range {
+    range_name    = var.cluster_secondary_range_name
+    ip_cidr_range = var.cluster_secondary_range_cidr
+  }
+  secondary_ip_range {
+    range_name    = var.services_secondary_range_name
+    ip_cidr_range = var.services_secondary_range_cidr
+  }
 
   # When enabled, VMs in this subnetwork without external IP addresses can
   # access Google APIs and services by using Private Google Access. This is
   # set explicitly to prevent Google's default from fighting with Terraform.
   private_ip_google_access = true
 
   depends_on = [
-    "google_compute_network.vpc_network",
+    google_compute_network.vpc_network,
   ]
 }
 
 module "cluster" {
-  source  = "jetstack/gke-cluster/google"
-  version = "0.1.0-beta2"
+  source  = "../terraform-google-gke-cluster"
+  #version = "0.2.0-alpha1"
```

Comment:
The source will need to go back to `jetstack/gke-cluster/google` before this is merged, with the version set to `0.2.0-alpha1`. I can then add a tag as soon as it's merged so that it's available.